checker.go

```diff
@@ -15,6 +16,19 @@ func cleanMsg(m map[string]interface{}) map[string]interface{} {
 	if _, ok := m["context"].(map[string]interface{}); ok {
 		m["context"] = delete(m["context"].(map[string]interface{}), "library")
 	}
+	// delete empty fields
+	for k, v := range m {
+		if reflect.TypeOf(v).String() != "string" {
```

Comment:
This seems a bit broad. Could we narrow this check so that the logic is `if isMap() { ... }`